packages/react-vis/package.json

```diff
@@ -27,9 +27,9 @@
     "build": "yarn run clean && babel --root-mode upward src -d dist --copy-files && BABEL_ENV=es babel --root-mode upward src -d es --copy-files && node-sass src/main.scss dist/style.css --output-style compressed && yarn run build:browser",
     "lint-styles": "stylelint src/styles/*.scss --syntax scss",
     "test:windows": "babel-node --inspect ./tests/index.js",
-    "test": "node --inspect ./node_modules/.bin/babel-node --root-mode upward ./tests/index.js --only='../showcase/**/*.js,src,tests'",
+    "test": "NODE_ENV=development jest",
```

Comment:
I had to add this because by default jest sets `NODE_END` to `test`, see [this issue](https://github.com/facebook/jest/issues/1654)

We can also add a `test` key to [babel.config.json](https://github.com/uber/react-vis/blob/master/babel.config.json) but since all the plugins are copy-pasted between each key I didn't want to add one more, it would make it more likely to forget a plugin somewhere when we update the file.

Reply:
good call... I need to take another pass to clean up that babel config a little bit.